RELEASE_NOTES.md

```diff
@@ -96,6 +102,12 @@ v1.0.0-alpha.xx
   feature.
   [#1088](https://github.com/OpenAssetIO/OpenAssetIO/issues/1088)
 
+- `createContext` does not call `createState` if manager is not capable
```

Comment:
[Fix]
```suggestion
- `Manager.createContext` does not call `ManagerInterface.createState` if manager is not capable
```

Reply:
[Consider] Is this more an improvement?

Reply:
https://github.com/OpenAssetIO/OpenAssetIO/pull/1141/commits/205e205d8dff12b385c4a18c5c5082ad38facc68

It's in the improvement section, I think github may have collapsed the diff inbetween the headings.

Reply:
Sorry my mistake!